packages/react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -200,7 +200,8 @@ const Checkbox = React.forwardRef(
         <label
           htmlFor={id}
           className={`${prefix}--checkbox-label`}
-          title={title}>
+          title={title}
+        >
```

Comment:
```suggestion
          title={title}>
```

I believe this is enforced by the prettier formatting rules in the repo

Reply:
then, should i rollback this line?

Reply:
Running `yarn format` from the root will run prettier against the whole codebase if you don't have prettier installed in your editor. I just did this and updated the PR so this should be good to go now. Sorry for the confusion!